src/pages/docs/infrastructure/deployment-targets/offline-package-drop.md

```diff
@@ -4,14 +4,14 @@ pubDate: 2023-01-01
 modDate: 2023-01-01
 title: Offline package drop
 description: Offline package drop deployment targets allow you to deploy your applications in the most restricted security environments where Tentacles cannot be used.
-navOrder: 60
+navOrder: 70
 ---
 
 The offline package drop deployment target makes it possible for Octopus to bundle all the files needed to perform a deployment to a deployment target, even when a direct connection to the deployment target isn't always possible, for instance, if a security policy, compliance control, or network topology make a direct connection impossible.
 
 You can treat the offline package drop just like any other target, but instead of the application being deployed Octopus will bundle up all of the files needed to perform the deployment on the *actual* target server.
 
-## Configuring the target {#OfflinePackageDrop-ConfiguringtheTarget}
+## Configuring the target {#target-configuration}
```

Comment:
These were updated in #2249 
I validated that they were not in use in any docs pages or in short.io